Libraries/xcsdk/Tools/xcrun.cpp

```diff
@@ -324,15 +327,27 @@ main(int argc, char **argv)
     /*
      * Determine the toolchains to use. Default to the SDK's toolchains.
      */
-    xcsdk::SDK::Toolchain::vector toolchains = target->toolchains();
-    if (!toolchain.empty()) {
+    xcsdk::SDK::Toolchain::vector toolchains;
+    if (!toolchainsInput.empty()) {
         /* If the custom toolchain exists, use it instead. */
-        if (auto TC = manager->findToolchain(toolchain)) {
-            toolchains = { TC };
-        } else {
-            fprintf(stderr, "error: unable to find toolchain '%s'\n", toolchain.c_str());
+        std::vector<std::string> toolchainTokens;
+        std::stringstream sstream(toolchainsInput);
+        copy(std::istream_iterator<std::string>(sstream),
+             std::istream_iterator<std::string>(),
+             std::back_inserter(toolchainTokens));
```

Comment:
You could use `pbxsetting::Type::ParseList()` here, it's a little shorter and IMO more clear about what it's doing.